Sources/RegexBenchmark/Suite/CssRegex.swift

```diff
@@ -0,0 +1,1672 @@
+import Foundation
+
+extension BenchmarkRunner {
+  mutating func addCSS() {
+    let r = "--([a-zA-Z0-9_-]+)\\s*:\\s*(.*?):"
+    
+    // sorry
+    let css = """
```

Comment:
Move this string into its own file and give it a better internal name (or make it a static var on a CSS benchmark type). We could have an `Inputs` folder for these kinds of things